src/common/imagpng.cpp

```diff
@@ -323,6 +384,18 @@ wxPNGHandler::LoadFile(wxImage *image,
     }
 
     png_read_image( png_ptr, lines );
+
+    // custom options
+    for (int textIndex = 0; textIndex < info_ptr->num_text; ++textIndex)
+    {
+        if (info_ptr->text[textIndex].compression == PNG_TEXT_COMPRESSION_NONE)
```

Comment:
Why do we ignore the ones with different compression? IMHO it at least merits a comment.